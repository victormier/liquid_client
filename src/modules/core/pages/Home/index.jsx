import React from 'react';
import { Grid, Row, Col } from 'react-flexbox-grid';
import { Link } from 'react-router';
import Button from 'components/common/Button';
import gridStyles from 'styles/base/grid.scss';
import logo from 'assets/images/logo.svg';
import styles from './styles.scss';

const Home = () => (
  <Grid fluid className={gridStyles.mainGrid}>
    <Row className={gridStyles.fullHeight}>
      <Row top="xs" className={gridStyles.fullWidth}>
        <Col xs={8} className={styles.logo}>
          <img alt="Logo" src={logo} />
          <h1>
            The simplest way<br />
            to take back control<br />
            of your cashflow
          </h1>
        </Col>
      </Row>
      <Row bottom="xs" className={gridStyles.fullWidth}>
        <Col xs={6}>
          <Link to="/login"><Button text="Log in" color="blue" /></Link>
        </Col>
        <Col xs={6}>
          <Link to="/signup">
            <Button text="Sign up" color="transparent" />
          </Link>
        </Col>
      </Row>
    </Row>
  </Grid>
);

export default Home;
