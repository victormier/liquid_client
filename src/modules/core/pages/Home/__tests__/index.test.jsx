import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import Home from '../index'


describe('component', () => {
  it('renders', () => {
    const container = shallow(
      <Home />
    )

    expect(shallowToJson(container)).toMatchSnapshot()
  })
})
