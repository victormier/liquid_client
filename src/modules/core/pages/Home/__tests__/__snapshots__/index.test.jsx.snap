// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component renders 1`] = `
<Grid
  fluid={true}
>
  <Row
    bottom="xs"
  >
    <Col
      xs={6}
    >
      <Link
        onlyActiveOnIndex={false}
        style={Object {}}
        to="/login"
      >
        <Button
          color="blue"
          text="Log in"
        />
      </Link>
    </Col>
    <Col
      xs={6}
    >
      <Button
        color="transparent"
        text="Sign up"
      />
    </Col>
  </Row>
</Grid>
`;
